app/views/planning_applications/validation/additional_document_validation_requests/_form.html.erb

```diff
@@ -14,11 +14,11 @@
       ) %>
 
       <%= form.govuk_text_field :document_request_type,
-       label: { text: 'Please specify the new document type:', size: 's' },
+       label: { size: 'm', class: "govuk-label govuk-label--m", text: 'Please specify the new document type:' },
        hint: { text: 'Eg. Existing floor plans' } %>
 
       <%= form.govuk_text_area :reason,
-       label: { text: 'Please specify the reason you have requested this document?', size: 's'},
+       label: { size: 'm', class: "govuk-label govuk-label--m", text: 'Please specify the reason you have requested this document?'},
```

Comment:
Same here - the `govuk-label` and `govuk-label--m` should be added automatically